### ISSUE TITLE
关于rm中lm loss计算的疑问

### ISSUE THREAD

**Issue by DZ9** [OpenLMLab/MOSS-RLHF#48]
在[reward_trainer.py](https://github.com/OpenLMLab/MOSS-RLHF/blob/main/rm/reward_trainer.py#L147)这里，删除了lm_logits中最后一个token的概率分布，但是在下面的label里面是删除了第一个词，想问下这里是怎么对应的呢
![image](https://github.com/OpenLMLab/MOSS-RLHF/assets/6756880/fe71ee16-ba46-4797-bce1-3160800976ed)

**Comment by refrain-wbh** [comment]
这是因为，logit是输出，而label or loss mask是输入。decoder架构（例如llama）预测next token，因此输入的token和输出的next token是对应关系。
